src/background/chrome-api.ts

```diff
@@ -200,7 +200,10 @@ export async function executeScript(
     return results[0].result;
   }
 
-  const result = await chromeAPI_.tabs.executeScript(tabId, { frameId, file });
+  const result = (await chromeAPI_.tabs.executeScript(tabId, {
+    frameId,
+    file,
+  })) as unknown[];
```

Comment:
Nor putting a lot of effort into this, as this is part of Manifest v2 and getting removed soon.